modules/virt-customizing-storage-profile-default-cloning-strategy.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/storage/virt-configuring-storage-profile.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-customizing-storage-profile-default-cloning-strategy_{context}"]
+= Setting a default cloning strategy by using a storage profile
```

Comment:
```suggestion
= Setting a default cloning strategy using a storage profile
```

Reply:
Our CNV docs tend to use "by using the web console." There are many instances of it in the content.